src/main/resources/assets/prefab/lang/tr_tr.json

```diff
@@ -92,17 +92,17 @@
     "block.prefab.block_paper_lantern": "Kağıt Fener",
     "block.prefab.block_compressed_obsidian": "Sıkıştırılmış Obsidyen",
     "block.prefab.block_double_compressed_obsidian": "Çifte Sıkıştırılmış Obsidyen",
-    "block.prefab.block_phasic": "Fazlık Blok",
+    "block.prefab.block_phasic": "Fazik Blok",
     "block.prefab.block_boundary": "Sınır Bloğu",
     "block.prefab.block_glass_stairs": "Cam Merdiven",
     "block.prefab.block_glass_slab": "Cam Basamak",
 
-    "block.prefab.block_dirt_stairs": "Kirli Merdivenler",
-    "block.prefab.block_grass_stairs": "Çim Merdivenler",
-    "block.prefab.block_dirt_slab": "Kir Levha",
-    "block.prefab.block_grass_slab": "çim levha",
-    "block.prefab.block_dirt_wall": "Kir Duvar",
-    "block.prefab.block_grass_wall": "çim duvar",
+    "block.prefab.block_dirt_stairs": "Toprak Merdivenler",
+    "block.prefab.block_grass_stairs": "Çimenli Toprak Merdivenler",
+    "block.prefab.block_dirt_slab": "Toprak Basamak",
+    "block.prefab.block_grass_slab": "Çimenli Toprak",
```

Comment:
This translates to "Grassy Soil" and is used for both the Grass Slab and Grass Wall.